### PR TITLE
Overview page displays Public datasets

### PR DESCRIPTION
**Pull request by MVarshini**
PBENCH-1309

Overview page displays Public datasets

### CODE REVIEW

File: dashboard/src/actions/overviewActions.js [hunk 1]
```diff
@@ -13,16 +12,13 @@ import { uriTemplate } from "../utils/helper";
...
-
-  const loggedIn = Cookies.get("isLoggedIn");
   try {
     if (alreadyRendered) {
       dispatch({ type: TYPES.LOADING });
     }
     const params = new URLSearchParams();
 
-    addParams(params, loggedIn, datasetType);
+    addParams(params);
```
Comment (dbutenhof) [inline dashboard/src/actions/overviewActions.js:21]: How does the `?mine` filter get added on the Overview page now??

I think this could have been solved by hardcoding "my datasets" `datasetType` in the call here, (assuming this is really used [only] for the Overview page?) whereas now I believe we're unconditionally getting all visible datasets?

**Review by dbutenhof — CHANGES_REQUESTED** [review]

**Review by webbnh — COMMENTED** [review]
The change looks good...except for the issue that Dave raised.

File: dashboard/src/actions/overviewActions.js (hunk 1 above)
Comment (MVarshini) [reply dashboard/src/actions/overviewActions.js]: Oops! I forgot the main part. 

Updated the code.

**Review by dbutenhof — APPROVED** [review]

File: dashboard/src/actions/datasetListActions.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ export const fetchDatasets = (page) => async (dispatch, getState) => {
       getState().datasetlist;
     const datasetType = getState().comparison.datasetType;
     let publicData = [...getState().datasetlist.publicData];
+
     const params = new URLSearchParams();
     addParams(params, loggedIn, datasetType);
     params.append("offset", offset);
```
Comment (webbnh) [inline dashboard/src/actions/datasetListActions.js:42]: The only change to this module now is the addition of this one blank line.  It would have been nice to revert this change to minimize the churn, but let's just merge and move on.
